sdk/appconfiguration/app-configuration/src/index.ts

```diff
@@ -91,6 +111,21 @@ export class AppConfigurationClient {
     }
   }
 
+  /**
+   * Add a setting into the Azure App Configuration service, failing if it 
+   * already exists.
+   * 
+   * Example usage:
+   * ```ts
+   * const key = "MyKey";
+   * const label = "MyLabel";
+   * const value = "MyValue";
+   * const result = await client.addConfigurationSetting(key, { label, value });
+   * ```
+   * @param key The name of the key.
+   * @param configSettings A configuration value.
+   * @param options Optional parameters for the value being added.
```

Comment:
The optional parameters are not for the value being added.
They are for the process of adding the value. Maybe reword this?

Reply:
I'll reword it. 

The reason I had that is, for whatever reason, 'label' is _also_ in the optional arguments (as well as in the main 'configSettings' argument as well).